PowerNSX.psm1

```diff
@@ -16299,6 +16299,165 @@ function New-NsxEdgeRedistributionRule {
     end {}
 }
 
+function Invoke-NsxEdgeHaFailover {
```

Comment:
Could you please use 4 spaces per indentation. This will bring it in-line with the rest of the module.